lib/service_actor/collectionable.rb

```diff
@@ -6,23 +6,42 @@
 # Example:
 #
 #   class Pay < Actor
-#     input :provider, in: ["MANGOPAY", "PayPal", "Stripe"]
+#     input :provider, inclusion: ["MANGOPAY", "PayPal", "Stripe"]
```

Comment:
This is a breaking change. But I think it's appropriate. Otherwise, need to write `in` for normal use, and `inclusion` for advanced use. @sunny What do you think?